ZividOpenCV/CaptureUndistortRGB/CaptureUndistortRGB.cpp

```diff
@@ -25,60 +29,46 @@ int main()
         std::cout << "Connecting to the camera" << std::endl;
         auto camera = zivid.connectCamera();
 
-        std::cout << "Configuring the camera settings" << std::endl;
-        camera << Zivid::Settings::Iris{ 21 } << Zivid::Settings::ExposureTime{ std::chrono::microseconds{ 20000 } }
-               << Zivid::Settings::Gain{ 1 } << Zivid::Settings::Brightness{ 1.0 };
-
-        std::cout << "Capturing a frame" << std::endl;
-        const auto frame = camera.capture();
-
-        std::cout << "Setting up visualization" << std::endl;
-        Zivid::CloudVisualizer vis;
-        zivid.setDefaultComputeDevice(vis.computeDevice());
-
-        std::cout << "Displaying the point cloud" << std::endl;
-        vis.showMaximized();
-        vis.show(frame);
-        vis.resetToFit();
-
-        std::cout << "Running the visualizer. Blocking until the window closes" << std::endl;
-        vis.run();
-
-        std::cout << "Converting ZDF point cloud to OpenCV format" << std::endl;
+        std::cout << "Enter <2d><Return> for 2D only, and just <Return> for 3D" << std::endl;
+        const auto command = getInput();
+        bool use2D = false;
+        if(command == "2d" || command == "2D")
```

Comment:
Instead of the blank for 3d I would add "3d" or "3D". It is more intuitive. Blank could work for one of them, or not, doesn't matter.

Reply:
It would work with 3d or whatever really. Stating it explicitly would be untrue.